substrate/frame/staking/src/lib.rs

```diff
@@ -1261,6 +1271,10 @@ impl<const DISABLING_LIMIT_FACTOR: usize> UpToThresholdDisablingStrategy<DISABLI
 impl<T: Config, const DISABLING_LIMIT_FACTOR: usize> DisablingStrategy<T>
 	for UpToThresholdDisablingStrategy<DISABLING_LIMIT_FACTOR>
 {
+	fn disable_threshold(validators_len: usize) -> usize {
+		Self::disable_threshold(validators_len)
```

Comment:
Ok, now I see where the name is coming from. Not a great name to be honest. If I knew nothing I would expect that this is some threshold that when hit we start disabling.

Reply:
yeah, a `disabling_limit` would be more accurate

Reply:
Fixed